src/main/java/org/opensearch/search/relevance/configuration/SearchConfigurationExtBuilder.java

```diff
@@ -124,8 +124,10 @@ public int hashCode() {
     return Objects.hash(this.getClass(), this.resultTransformerConfigurations);
   }
 
-  public void setResultTransformers(final List<ResultTransformerConfiguration> resultTransformerConfigurations) {
-    this.resultTransformerConfigurations = resultTransformerConfigurations; }
+  public SearchConfigurationExtBuilder setResultTransformers(final List<ResultTransformerConfiguration> resultTransformerConfigurations) {
```

Comment:
is there a different standard now for creating a "set" method that returns something? Should this be named differently?

Reply:
I normally write builders with `withXXX` methods for chaining, but the longstanding convention with builders in the codebase uses `setXXX` that returns the builder.

I disagree with the convention, but I need to follow it.